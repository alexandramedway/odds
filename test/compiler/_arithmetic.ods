state 1 + 2.1
state 1 - 2.1
state 1 * 2.1
state 1 / 2.1
state 1 % 2.1
state 1 ** 2.1
state -42
state 42
state 1 + -43
state 1 * 2 + 3 ** 4
state 1 / 2 % 3 % 4
state 1 + 2 - 3 / 4
state 1 * (2 + 3)
state print(1 + 2.1)
state print(1 - 2.1)
state print(1 * 2.1)
state print(1 / 2.0)
state print(1 % 2.1)
state print(1 ** 2.1)
state print(-42)
state print(42)
state print(1 + -43)
state print(1 * 2 + 3 ** 4)
state print(1 / 2 % 3 % 4)
state print(1 + 2 - 3 / 4)
state print(1 * (2 + 3))
