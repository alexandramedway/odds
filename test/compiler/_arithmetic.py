(1 + 2.1)
(1 - 2.1)
(1 * 2.1)
(1 / 2.1)
(1 % 2.1)
(1 ** 2.1)
(-42)
42
(1 + (-43))
((1 * 2) + (3 ** 4))
(((1 / 2) % 3) % 4)
((1 + 2) - (3 / 4))
(1 * (2 + 3))
print((1 + 2.1))
print((1 - 2.1))
print((1 * 2.1))
print((1 / 2.1))
print((1 % 2.1))
print((1 ** 2.1))
print((-42))
print(42)
print((1 + (-43)))
print(((1 * 2) + (3 ** 4)))
print((((1 / 2) % 3) % 4))
print(((1 + 2) - (3 / 4)))
print((1 * (2 + 3)))
