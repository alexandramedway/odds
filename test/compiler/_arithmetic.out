3.1
-1.1
2.1
0.47619047619047616
1.0
1.0
-42
-42
42
83
0
3
5
